templates/terraform/custom_import/compute_network_delete_default_route.erb

```diff
@@ -0,0 +1,3 @@
+// Explicitly set to default as a workaround for `ImportStateVerify` tests, and so that users
+// don't see a diff immediately after import.
+d.Set("delete_default_route", false)
```

Comment:
I think we're going to need to use a pattern similar to `default_if_empty` (https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/custom_flatten/default_if_empty.erb) in `Read` (probably in a decoder?); as-is, this will cause a diff for all existing users of the resource. Doing this in `Read` will supersede the import-only behaviour, but we could keep it for clarity / for the sake of being explicit if you'd like.

Reply:
Added the decoder, I think this is what you mean, please let me know otherwise.